pygerduty/v2.py

```diff
@@ -71,11 +71,38 @@ def create(self, **kwargs):
         if "requester_id" in kwargs:
             extra_headers["From"] = kwargs.pop("requester_id")
 
-        data[self.sname] = kwargs
+        new_kwargs = {}
+
+        for kwarg_key, kwarg_value in kwargs.iteritems():
+            if kwarg_key.endswith('_id'):
+                new_kwargs = Collection.id_to_obj(kwarg_key, kwarg_value)
+            elif kwarg_key.endswith('_ids'):
+                new_kwargs = Collection.ids_to_objs(kwarg_key, kwarg_value)
+            else:
+                new_kwargs[kwarg_key] = kwarg_value
+        data[self.sname] = new_kwargs
 
         response = self.pagerduty.request("POST", path, data=_json_dumper(data), extra_headers=extra_headers)
         return self.container(self, **response.get(self.sname, {}))
 
+    @staticmethod
+    def id_to_obj(key, value):
+        new_key = key[:-3]
+        if key.endswith('_ids'):
+            new_key = key[:-4] + "s"
```

Comment:
Are there any resources that end in `y` that might be `ies` instead of `s`? e.g. `policy_ids` -> `policies`. I believe we have a `pluralize` function that could be reused?